toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -71,133 +152,236 @@ func CustomizeImageWithConfigFile(buildDir string, configFile string, imageFile
 	return nil
 }
 
+func cleanUp(ic *ImageCustomizerParameters) error {
+
```

Comment:
There shouldn't be an empty newline after `{`.

Reply:
pending.